config/settings.yml

```diff
@@ -86,12 +86,10 @@ sign_in:
 
 terms_of_use:
   current_version: v1
-  provisioner_cookie_domain: localhost
```

Comment:
Riley added this last year in https://github.com/department-of-veterans-affairs/vets-api/issues/14455. @rileyanderson, we don't see `Settings.terms_of_use.provisioner_cookie_domain` called anywhere. I just wanted to check with you before we deleted this.

Reply:
Talked to Riley and `provisioner_cookie_domain` isn't being used